docs/index.md

```diff
@@ -27,13 +27,37 @@ can be used to:
 
 ## Instructions
 
-1.  Set up Eclipse and CLDR, according to instructions:
+### General Setup for Maven
+
+1.  Configure Maven settings (including Github tokens) according to these instructions:
     <http://cldr.unicode.org/development/maven>
+2.  Get your github account authorized for <https://github.com/unicode-org/unicodetools>,
+    create a fork under your account, and create a local clone.
+3.  Clone both the [Unicode Tools](https://github.com/unicode-org/unicodetools) and [CLDR](https://github.com/unicode-org/cldr) repositories:
+```
+git clone https://github.com/unicode-org/unicodetools.git
+git clone https://github.com/unicode-org/cldr.git
+```
+4.  In the root of the `unicodetools` local working copy, create the `Generated` folder 
```

Comment:
I added notes for in-source and out-of-source build setups. I tested out both sets of setup & task commands, and they run without errors locally.

FYI @srl295 I also deduced that the `mvn ... compile` step is somehow performing some sort of pre-requisite initialization step.  I'm not sure how it is the case that the `mvn ... compile` command has to be run first before the other commands should be run (including a `mvn ... package` command), since `compile` is a precursor to `package` in the Maven lifecycle progression whenever `package` is run. My naive suspicion is that there are some settings for <repositories> in that `.github/worklfows/mvn-settings.xml` that could (should?) be moved up to the parent `pom.xml`. We don't need to solve it here, though, so just FYI & food for thought.